examples/python/cluster_example.py

```diff
@@ -0,0 +1,125 @@
+import asyncio
+from typing import List, Tuple
+
+from glide import (
+    AllNodes,
+    ClosingError,
+    GlideClusterClient,
+    GlideClusterClientConfiguration,
+    InfoSection,
+    Logger,
+    LogLevel,
+    NodeAddress,
+    RequestError,
+)
+
+
+async def create_client(
+    nodes_list: List[Tuple[str, int]] = [("localhost", 47243)]
+) -> GlideClusterClient:
+    """
+    Creates and returns a GlideClusterClient instance.
+
+    This function initializes a GlideClusterClient with the provided list of nodes.
+    In cluster mode, add the address of any node; the client will automatically
+    discover all nodes in the cluster.
+
+
+    Args:
+        nodes_list (List[Tuple[str, int]]): A list of tuples where each tuple
+            contains a host (str) and port (int). Defaults to [("localhost", 6379)].
+
+    Returns:
+        GlideClusterClient: An instance of GlideClusterClient connected to the specified nodes.
```

Comment:
This conflicts with your previous comment (the comment that talks about cluster discovery)